src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -120,6 +122,20 @@ public int updateIsFailed(UUID id) {
         );
     }
 
+    /**
+     * Retrieve a list of letters which are not failed but still not printed.
+     *
+     * @return list of not printed letters until yesterday's 5pm
+     */
+    public List<NotPrintedLetter> getNotPrintedUntilYesterday() {
+        return jdbcTemplate.query(
+            "SELECT id, message_id, service, type, created_at, sent_to_print_at "
+                + "FROM letters "
+                + "WHERE is_failed = false AND printed_at IS NULL AND sent_to_print_at IS NOT NULL"
```

Comment:
One query here we have defined column type of `is_failed` as BOOLEAN type but haven't specified default value as FALSE.I imagine it will null in that case and above query needs to be updated?

Reply:
As per migration script column does not have a default value defined. So it is a boolean false. There will be a functional test proving that once plugged all this into service. For now can "trust" it is stored as `false` from this [test](https://github.com/hmcts/send-letter-producer-service/blob/master/src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/GetLetterStatusTest.java#L76-L88)